tests/Unit/Domain/ValueObjects/ShoppingItemFilterEnumTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Tests\Unit\Domain\ValueObjects;
+
+use Lindyhopchris\ShoppingList\Domain\ValueObjects\ShoppingItemFilterEnum;
+use PHPUnit\Framework\TestCase;
+
+class ShoppingItemFilterEnumTest extends TestCase
+{
+    public function testItGetsAllItems(): void
```

Comment:
Comment: these are definitely the three test scenarios you need to test, so good job on identifying them.